### PR TITLE
Testing out Github's new PR review stuff

### PR DESCRIPTION
**Pull request by esoergel**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -59,6 +61,8 @@ Once you have completed the above steps, you can use Docker to build and run all
 
 ### Set up your django environment
 
+Here is a tipo.
```
Comment (esoergel) [inline README.md]: *typo

File: docs/index.rst [hunk 2]
```diff
@@ -13,23 +13,25 @@ Welcome to CommCareHQ's documentation!
 	Here's a sample well-organized toc:
 	https://github.com/kennethreitz/python-guide/blob/master/docs/contents.rst.inc
 
+..
+  comment comment comment
```
Comment (esoergel) [inline docs/index.rst]: This comment is useless

Comment (esoergel) [inline docs/index.rst]: What does this new conversation look like?

**Review by esoergel — COMMENTED** [review]
Overall looks good!

File: docs/index.rst (hunk 2 above)
Comment (millerdev) [inline docs/index.rst]: threaded comments?

Comment (benrudolph) [inline docs/index.rst]: it didn't give me the option to start a review 😢

Comment (benrudolph) [inline docs/index.rst]: now i see how to do it

**Review by benrudolph — CHANGES_REQUESTED** [review]
Dog you gotta change that before merging

**Review by millerdev — CHANGES_REQUESTED** [review]
the code won't work unless these typos are fixed.

**Review by millerdev — APPROVED** [review]

**Comment by esoergel** [comment]
Weird, I can't reply to @millerdev 's comment "threaded comments?".  I wonder if that's because I'm the PR author.  I also dont' have the option to request changes on my own PR.

File: docs/index.rst [hunk 3]
```diff
@@ -13,23 +13,25 @@ Welcome to CommCareHQ's documentation!
...
    class_views
+   ui_helpers
    forms
    management_commands
-   commtrack
    cloudcare
    translations
    profiling
    elasticsearch
+   commtrack
```
Comment (czue) [inline docs/index.rst]: :skull:

File: README.md (hunk 1 above)
Comment (czue) [inline README.md]: should address

**Review by czue — CHANGES_REQUESTED** [review]
still some outstanding problems

**Comment by czue** [comment]
neat

**Pull request closed without merging**